services/httpd/response_logger.go

```diff
@@ -99,7 +101,7 @@ func buildLogLine(l *responseLogger, r *http.Request, start time.Time) string {
 
 	userAgent := r.UserAgent()
 
-	fields := []string{
+	return fmt.Sprintf("%s %s %s [%s] \"%s %s %s\" %s %s \"%s\" \"%s\" %s %s",
```

Comment:
The second '%s' is a static "-", so that can be included directly. The start time is still being sent in as a result of `fmt.Sprintf` itself... making the actual output double-bracketed. The `strconv` calls are now unnecessary, and should be replaced with `%d` verbs.